content/en/docs/release-oversight/payload-testing.md

```diff
@@ -1,169 +1,80 @@
 
+For cost reasons, we cannot run every possible configuration against pull requests pre-merge. Payload testing, however, batches multiple changes together several times a day and allows us to run extensive testing. This provides an excellent secondary layer to catch regressions that may have merged but have not made it into an accepted payload.
 
-## Usage
+## Payload / Job Organization
 
-### Payload Command
-Any collaborator of the GitHub OpenShift organization can issue the command on a pull request to a branch of a repository of the organization that contributes to OpenShift official images:
+The Release Controller maintains payloads for multiple architectures, each of which has its own status page. At present, TRT primarily focuses on [amd64](https://amd64.ocp.releases.ci.openshift.org/).
 
-> /payload <ocp_version> <ci|nightly> <informing|blocking>
+### CI vs. Nightly Payloads
 
-For example, if `/payload 4.10 nightly informing` is issued on a PR, the robot will reply the list of the triggered jobs:
+You will see payloads broken down by OpenShift Release and CI and nightly streams.
 
-![payload command](/payload-cmd.png)
+CI payloads largely track master branches and are built directly in our CI system.
 
-The jobs triggered by the command are determined by [OpenShift Release Controllers](/docs/getting-started/useful-links/#services).
-The linked page from [payload-tests portal](https://pr-payload-tests.ci.openshift.org/runs/) at the bottom of the comment shows the status of the payload testing and the details of those jobs.
+Nightly payloads are more official as they are built using the full Red Hat build infrastructure. These payloads are candidates for EC/release selection, and thus you will see considerably more testing run. This payload stream is susceptible to outages and delays in the build system. Despite its name, nightly payloads are typically run 2-4 times a day.
 
-The images from multiple PRs can also be included in the payload tested by using the `/payload-with-prs` command, such as
-> /payload-with-prs <ocp_version> <ci|nightly> <informing|blocking> <org/repo#number> [<org/repo#number ...]
+### Blocking vs. Informing Jobs
 
-A particular job or set of jobs can be triggered by `/payload-job`, such as
+Jobs in the blocking section will reject the payload if they fail. Informing jobs will not. We need to be careful with what goes into blocking for this reason. See [Extending OpenShift Release Gates](https://docs.ci.openshift.org/docs/architecture/release-gating/) for more on how a job can become blocking.
 
-> /payload-job <periodic_ci_openshift_release_some_job> <periodic_ci_openshift_release_another_job>
+## Aggregated Jobs
```

Comment:
This now includes some of the content from "Improving our CI Signal" doc, I did not remove it though, I'm not sure what to do with that page. But I did want an overview here of what goes on in payload testing to include aggregation.